src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -133,6 +135,8 @@ public boolean compile(String cp) throws IOException, InterruptedException {
         compileBuilder.command().add("-Dsvm.platform=org.graalvm.nativeimage.Platform$"+jniPlatform);
         compileBuilder.command().add("-cp");
         compileBuilder.command().add(classPath);
+        if (compilerArgs != null)
+            compileBuilder.command().add(compilerArgs);
```

Comment:
`compilerArgs` is a string, so only one argument will be allowed.
If more arguments are required, usually those can be added with a comma separated string. However I believe that won't work for the process builder. 
There is a convenient `Strings::split` method that could be used in this case, so the user's options will be added as a list.

Reply:
My idea was that user could add space separated arguments (since ProcessBuilder would  eventually join all arguments into a single string anyway - through [ProcessImpl](https://github.com/frohoff/jdk8u-jdk/blob/da0da73ab82ed714dc5be94acd2f0d00fbdfe2e9/src/windows/classes/java/lang/ProcessImpl.java#L193))

However that on retrospect really seems hacky so I changed compilerArgs to be a list of strings in later commits.